rll/collectd.sh

```diff
@@ -7,6 +7,11 @@
 #   for aggregating and displaying monitoring data. Collectd sends its monitoring
 #   data to the RightLink process on the localhost as HTTP using the write_http plugin.
 #   RightLink then forwards that data to the TSS servers over HTTPS with authentication.
+#
+# ## Known Limitations:
+# Running this script on very small instance types (less than 1GB memory) may result in a failure
+# to install "collectd_tcp_network_connect" because of a lack of memory.
+# This can be worked around by adding a 1GB or larger swap file to your server prior to running this script.
```

Comment:
In order to be parsed correctly as part of the description, this paragraph needs to be indented to the same level as the previous paragraph.